packages/docusaurus-plugin-content-pages/src/pluginOptionSchema.ts

```diff
@@ -20,12 +20,14 @@ export const DEFAULT_OPTIONS: PluginOptions = {
   remarkPlugins: [],
   rehypePlugins: [],
   admonitions: {},
+  exclude: ['**/_*.{js,jsx,ts,tsx,md,mdx}'],
```

Comment:
wonder if it's easy to exclude by default `.test.{js,jsx,ts,tsx}` + content of `__tests__` folder?

Reply:
I will add it